akka-stream-tests/src/test/scala/akka/stream/io/StreamConvertersAsInputStreamErrorSpec.scala

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (C) 2019 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.stream.io
+
+import java.io.{IOException, InputStream}
+import java.util.concurrent.ThreadLocalRandom
+
+import akka.stream._
+import akka.stream.scaladsl.{Source, StreamConverters}
+import akka.stream.testkit.Utils._
+import akka.stream.testkit._
+import akka.util.ByteString
+import akka.NotUsed
+
+/**
+ * https://github.com/akka/akka/issues/25614
+ * Behavior does not change if you use Supervision.stoppingDecider or not, or
```

Comment:
Only a subset of the operators uses the supervision strategy attribute (and those should have explicit docs saying so), I think for this one it does not make much sense as it is a sink, if it sees a failure that means the upstream has already failed, so not much it can do with any strategy.